### ISSUE TITLE
Unable to compile with io.js

### ISSUE THREAD

**Issue by angeloashmore** [firebase/blaze_compiler#52]
When using io.js running `blaze rules.yaml -v` outputs `undefined` and does not create a `rules.json` file.

**Console output**:

~~~ shell
# npm run build => blaze rules.yaml -v

$ npm run build

> <my-app-name>@0.0.1 build /<my-app-dir>
> blaze rules.yaml -v

args { _: [ 'rules.yaml' ],
  v: true,
  java: false,
  '$0': '/<my-app-dir>/node_modules/.bin/blaze' }
transpiling rules.yaml
undefined
~~~

`<my-app-name>` is the name of my app removed for privacy.
`<my-app-dir>` is the directory to my app removed for privacy.

`rules.yaml` is the same as in the README.md.

Any ideas on what's happening?

**Comment by tomlarkworthy** [comment]
I tried a fresh install and I can compile the examples. Can you try running it on https://github.com/firebase/blaze_compiler/blob/master/examples/mail_example.yaml ?

**Comment by angeloashmore** [comment]
1. Fresh install of blaze_compiler
2. New dir with mail_example.ml
3. `blaze rules.yaml`

I got the same output as above. I am using io.js rather than node. Could that cause any issues?

**Comment by angeloashmore** [comment]
Yup, io.js seems to be incompatible with blaze. I was able to successfully compile the rules with node.

I'll leave this open, but feel free to close this if io.js is not a priority.

**Comment by tomlarkworthy** [comment]
Oh, thanks for letting me know what the problem was

**Comment by nicholasf** [comment]
I just ran into the same issue.

**Comment by tomlarkworthy** [comment]
Hmmm, maybe its a new problem as this is the first time this has been brought up.

**Comment by tomlarkworthy** [comment]
fixed in 0.0.33
